### ISSUE TITLE
Webview version/UserAgent on macOS?

### ISSUE THREAD

**Issue by retorquere** [r0x0r/pywebview#541]
### Specification
- pywebview version: 3.2
- platform / version: macOS 10.15.5

### Description  
  
When I open https://outlook.office.com/calendar/view/day, it shows the legacy UI of outlook, which means it thinks the newer, javascript-heavy UI is not supported. This may be because pywebview uses an older cocoa component, or it sets an unknown (to outlook) user agent.
  
### Practicalities

- YES I am willing to work on this issue myself

- YES I am prepared to support this issue financially.

**Comment by r0x0r** [comment]
There is an ability to change a user agent in the current master. You may try that and see if Outlook serves you a different version. Look for the closed user agent issue for details.

**Comment by retorquere** [comment]
Yep, that does the trick. Is there a timeline for this to be released?

**Comment by r0x0r** [comment]
Hopefully this month.
